assets/js/admin/course-pricing-promo-sidebar.js

```diff
@@ -0,0 +1,60 @@
+import { __ } from '@wordpress/i18n';
+import { applyFilters, addFilter } from '@wordpress/hooks';
+import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
+import { escapeHTML } from '@wordpress/escape-html';
+import { ExternalLink } from '@wordpress/components';
+
+/**
+ * Course Pricing Promo Sidebar component.
+ */
+
+const CoursePricingPromoSidebar = () => {
+	/**
+	 * Filters to get description for pricing component.
+	 *
+	 * @since 4.1.0
+	 *
+	 * @hook  senseiCoursePricingDescription
+	 * @return {Object}
+	 */
+	const description = applyFilters( 'senseiCoursePricingDescription', {
+		text: __(
+			'Sell this course using WooCommerce - integrates with subscriptions, memberships, affiliates, and more.'
+		),
+		url:
+			'https://senseilms.com/pricing/?utm_source=plugin_sensei&utm_medium=upsell&utm_campaign=course_pricing',
+	} );
+
+	return (
+		<PluginDocumentSettingPanel
+			name="sensei-course-theme"
+			title={ <>{ __( 'Pricing ', 'sensei-lms' ) }</> }
+		>
+			<div className="sensei-pricing-promo">
+				<p> { escapeHTML( description.text ) } </p>
+				<ExternalLink href={ description.url }>
+					{ __( 'Upgrade to Sensei Pro', 'sensei-lms' ) }
+				</ExternalLink>
+				<p className="sensei-pricing-promo__upgrade-content-text">
+					{ __(
+						'To access this course, learners will need to purchase one of the assigned products.',
+						'sensei-lms'
+					) }
+				</p>
+				<div className="sensei-pricing-promo__upgrade-new-course">
+					<p className="sensei-pricing-promo__upgrade-new-course-text">
+						{ __(
+							"You don't have any products yet. Get started by creating a new WooCommerce product",
+							'sensei-lms'
+						) }
+					</p>
+					<div className="sensei-pricing-promo__upgrade_new_course_mock_button">
+						{ __( 'Create a product', 'sensei-lms' ) }
```

Comment:
This button is a bit wide in the UI. Could we shorten it? Also, just curious if you looked into using the actual `Button` component here, and if so, were there any particular challenges with it that caused you to switch to a `div`?

Reply:
Not a real reason, but because I wanted to make it more as static mock content, this was also before I moved it to be component. I've changed it to disabled Button.

Reply:
Here is changed to Button, also I removed bunch of CSS, was redundant: [link](https://github.com/Automattic/sensei/pull/4810/files#diff-0cdf94ac1cfb0a1e90adaef288fdb975f07f4c39ff34e0de2003256181b3ec93R55)